$NetBSD$

Change hardwired install location to be under ${PREFIX}.  weewx
expects to have a subprefix where it stores programs, data, and
generates html.  This is not aligned with packaging norms, but at
least for now, do not try to change that.

--- setup.cfg.orig	2019-05-31 23:47:14.000000000 +0000
+++ setup.cfg
@@ -6,7 +6,7 @@
 # windows: c:\Users\weewx
 
 [install]
-home = /home/weewx
+home = @PREFIX@/weewx
 
 prefix =
 exec-prefix =
